Remotion/Core/Core/Context/SafeContext.Task.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+// We must use an alias as the class name is the same
+using Task_ = System.Threading.Tasks.Task;
```

Comment:
I think `SystemTask` would be better when reading the code without needing to check where the `_` is coming from. Goes for the other ones, too.